tensorflow/compiler/xla/stream_executor/rocm/hip_blas_lt.h

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 #include "rocm/rocm_config.h"
 #if TF_HIPBLASLT
 
-#if TF_ROCM_VERSION < 50700
+#if TF_ROCM_VERSION < 60000
 #define hipblasltDatatype_t hipblasDatatype_t
```

Comment:
Does it make sense to only do the remapping in this header file and use it as an include in other places where said remapping is needed?

Reply:
yes, @pemeliya is working on a better fix. This change is to specifically designate the mapping for rocm 6.0 and later (not 5.7 and later)